content/learning-paths/servers-and-cloud-computing/cca-trustee/flow.md

```diff
@@ -0,0 +1,283 @@
+        error_type: "https://github.com/confidential-containers/kbs/errors/TokenVerifierError",
+        detail: "Token Verifier error",
+    }
+[2025-07-23T14:42:55Z WARN  kbs_protocol::client::token_client] Authenticating with KBS failed. Get a new token from the token provider: ErrorInformation {
+        error_type: "https://github.com/confidential-containers/kbs/errors/TokenVerifierError",
+        detail: "Token Verifier error",
+    }
+Error: request unauthorized
+```
+
+#### Evaluate the Attestation Result
+
+In this step, you will examine the attestation result and see how it can be used to evaluate the trustworthiness of a CCA realm.
+
+The following command will use the `arc` tool to verify the cryptographic signature on the attestation result and display the result in a human-readable format:
+
+```bash
+./arc verify ear.jwt
+```
+
+{{% notice EAR expiry note %}}
+The EAR message produced by Trustee AS in this Learning Path demo is valid for 30 minutes.
+If you spend more time on anylising the message you will start seing errors from `arc verify` command:
+
+``` output
+Using JWK key from JWT header
+Error: verifying signed EAR from "ear.jwt" using "JWK header" key: failed verifying JWT message: jwt.Parse: failed to parse token: jwt.Validate: validation failed: "exp" not satisfied: token is expired
+```
+
+Please obtain a new EAR message by re-runing the attestation command.
+{{% /notice %}}
+
+
+`arc verify`  command produces quite a lot of output.
+However, the main part is the CCA attestation token similar to the one you inspected in
+[Get Started with CCA Attestation and Veraison](/learning-paths/servers-and-cloud-computing/cca-veraison) Learning Path.
+
+The most interesting part of the output is towards the bottom, and should look like this:
+
+```output
+[trustworthiness vectors]
+submod(cpu0):
+Instance Identity [none]: no claim being made
+Configuration [none]: no claim being made
+Executables [warning]: unrecognized run-time
+File System [none]: no claim being made
+Hardware [affirming]: genuine
+Runtime Opaque [none]: no claim being made
+Storage Opaque [none]: no claim being made
+Sourced Data [none]: no claim being made
+```
+
+This part of the output shows how the attestation service has compared the attestation token against its expectations of a trustworthy system.
+It also shows the conclusions that were drawn from that comparison.
+
+Please notice these two trustworthiness vectors in the result:
+- __Hardware [affirming]__. Evidence in the attestation token shows a good match against the expectations of CCA platform.
+- __Executables [warning]__. Attestation token does not show a good match against the expectations of a recognized genuine set of approved executables have been loaded during the boot process.
+
+You can also check the status of the EAR:
+```bash { output_lines = "2" }
+./arc verify ear.jwt |grep ear.status
+            "ear.status": "warning",
+```
+
```

Comment:
Again, it feels like we just trail off here. We need a closing sentence to propel the reader to the next section, something like:

The warning status is the reason why the KBS chose not to grant access to the secret that you requested in the earlier step. It has not concluded that the realm is trustworthy. But this is simply because we have not supplied an expected reference measurement for the realm. You will do this in the next step.